tests/run_tests.sh

```diff
@@ -2,7 +2,8 @@
 
 ###
 # Script requirments:
-#  apt-get install -y python-yaml virtualenv git
+apt-get install -y python-yaml virtualenv git
```

Comment:
No, everyone is expected to install this on his own. This would require running the script as root.